fvm/evm/handler/addressAllocator.go

```diff
@@ -12,15 +12,9 @@ import (
 const (
 	ledgerAddressAllocatorKey = "AddressAllocator"
 	uint64ByteSize            = 8
-	addressPrefixLen          = 12
-)
-
-var (
-	// prefixes:
-	// the first 12 bytes of addresses allocation
-	// leading zeros helps with storage and all zero is reserved for the EVM precompiles
-	FlowEVMPrecompileAddressPrefix = [addressPrefixLen]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}
-	FlowEVMCOAAddressPrefix        = [addressPrefixLen]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2}
+	// addressIndexShuffleSeed is used for shuffling address index
+	// shuffling index is used to make address postfixes look random
+	addressIndexShuffleSeed = uint64(0xFFEEDDCCBBAA9987)
```

Comment:
II suggest to rename `shuffling` to `mapping` since the function isn't shuffling in this case (we spoke about possibly shuffling the bits but we finally chose a different function)